### ISSUE TITLE
Parallax space is causing weird graphics issues.

### ISSUE THREAD

**Issue by Tacolizard** [tgstation/tgstation#22703]
In ooc i heard a player saying that space turned white. For me, parallax at any setting causes only one layer of parallax to display, making it look like there's barely any stars. If it helps, I run Linux and use a Windows 10 VM to play the game. i experience no issues with vgstation's parallax. my vm software is Oracle Virtualbox

NEW INFO

Oldman Robustin was having the same issue i was. It's not a VM thing. he says he got it on two comps:
Both are Windows 10
Neither uses VM
One is integrated graphics, the other is a GTX 970

basically, fix yo shit guys. probably a common issue.

**Comment by optimumtact** [comment]
This is probably due to the software rendering in a vm @MrStonedOne might know for sure

**Comment by Armhulen** [comment]
might for sure :thinking:

**Comment by Tacolizard** [comment]
some more info: i took a look at the parallax sprites in dream maker and only the bottom layer appears to contain stars, the same sparse amount i see in-game. However the 2nd layer sprites are all black with no stars.

**Comment by MrStonedOne** [comment]
@Tacolizard you were too zoomed out to see them

**Comment by optimumtact** [comment]
also >probably a common issue

yeah no

**Comment by lordpidey** [comment]
For what it's worth, I have the same issue on a windows 10 machine.  Though it's possibly due to having 2x video cards fighting for control.

**Comment by optimumtact** [comment]
see #22894

Multi GPU users is a seperate byond issue with an issue report open
